crates/bevy_pbr/src/prepass/mod.rs

```diff
@@ -265,24 +275,18 @@ type PreviousMeshFilter = With<Mesh3d>;
 #[cfg(feature = "meshlet")]
 type PreviousMeshFilter = Or<(With<Mesh3d>, With<MeshletMesh3d>)>;
 
+/// A run condition for [`update_mesh_previous_global_transforms`] that checks if any camera is active.
+pub fn any_camera_active(views: Query<&Camera, Or<(With<Camera3d>, With<ShadowView>)>>) -> bool {
```

Comment:
Do we have somewhere to have common run conditions? I feel like this could be useful for more than just the prepass module